src/services/videomania/index.jsx

```
@@ -416,6 +418,13 @@ export const VideoManiaPlayer = ({
                   : `${totalComments} comments`}
               </button>
             )}
+            <button
+              onClick={() => copyLink(video?.video)}
+              className={`action-button hover:${textColor}`}
+              disabled={isLoading}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)